TicTacToe/GamePlay.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+
+namespace TicTacToe
+{
+    public class GamePlay
+    {
+        private int _player;
+        private GameStatus _gameStatus;
+        private readonly Board _board;
+        private readonly int[,] _boardArray;
+        private bool _givenUp;
```

Comment:
your UserInputConsole also has a _givenUp or similar named bool, so I am wondering if we need this _givenUp bool in GamePlay